From 7bafcb1b39b03110ba5a375635d7b6b5ecf98bc4 Mon Sep 17 00:00:00 2001
Message-Id: <7bafcb1b39b03110ba5a375635d7b6b5ecf98bc4.1526409254.git.jan.steffens@gmail.com>
In-Reply-To: <a9554de2a6028540fa12a7ebdab4daf2b46010fa.1526409254.git.jan.steffens@gmail.com>
References: <a9554de2a6028540fa12a7ebdab4daf2b46010fa.1526409254.git.jan.steffens@gmail.com>
From: "Jan Alexander Steffens (heftig)" <jan.steffens@gmail.com>
Date: Sat, 14 Apr 2018 18:49:14 +0200
Subject: [PATCH 3/4] meson: Version libMesaOpenCL like autotools does

This is for parity with autotools. It names the library
libMesaOpenCL.so.1.0.0 and points mesa.icd to the .1 symlink.

opencl_version now matches configure.ac's OPENCL_VERSION.

Signed-off-by: Jan Alexander Steffens (heftig) <jan.steffens@gmail.com>
---
 src/gallium/targets/opencl/meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gallium/targets/opencl/meson.build b/src/gallium/targets/opencl/meson.build
index bebe0547d4..317ad8dab4 100644
--- a/src/gallium/targets/opencl/meson.build
+++ b/src/gallium/targets/opencl/meson.build
@@ -20,7 +20,7 @@
 
 opencl_link_args = []
 opencl_link_deps = []
-opencl_version = '1.0'
+opencl_version = '1'
 
 if with_ld_version_script
   opencl_link_args += [
@@ -55,7 +55,7 @@ libopencl = shared_library(
     cpp.find_library('clangLex', dirs : llvm_libdir),
     cpp.find_library('clangBasic', dirs : llvm_libdir),
   ],
-  version : opencl_version,
+  version : '@0@.0.0'.format(opencl_version),
   install : true,
 )
 
-- 
2.17.0

